dt-posts/custom-post-type.php

```diff
@@ -24,7 +24,7 @@ public function __construct( string $post_type, string $singular, string $plural
         add_filter( 'desktop_navbar_menu_options', [ $this, 'add_navigation_links' ], 20 );
         add_filter( 'dt_nav_add_post_menu', [ $this, 'dt_nav_add_post_menu' ], 10, 1 );
         add_filter( 'dt_templates_for_urls', [ $this, 'add_template_for_url' ] );
-        add_filter( 'dt_get_post_type_settings', [ $this, 'dt_get_post_type_settings' ], 10, 4 );
+        add_filter( 'dt_get_post_type_settings', [ $this, 'dt_get_post_type_settings' ], 1000, 4 ); // Called last within chain to ensure custom updates are applied last.
```

Comment:
@kodinkat this applies to all post types.
The number is low, so other hooks can add on to this one.
Why should it be 1000?

Reply:
To ensure custom updates are applied at the end of hooks execution chain. Number now reduced to something more sensible, and `contacts` and `groups` `base-setup` classes updated to also check for new updates.

Reply:
@kodinkat the custom post type filter needs to be loaded before the others, so that contacts, groups, etc can build upon it. Are there certain settings that are being overwritten? 
We can add a second `dt_get_post_type_settings` hook with a high value if needed.

Reply:
I added a later filter that will get the labels correctly. I see why that was tricky. Thanks!